spring-boot-admin-samples/spring-boot-admin-sample-servlet/src/main/java/de/codecentric/boot/admin/SecuritySecureConfig.java

```diff
@@ -31,10 +35,21 @@
 public class SecuritySecureConfig extends WebSecurityConfigurerAdapter {
     private final AdminServerProperties adminServer;
 
+    @Value("${spring.security.remember-me.timeout:1209600}")
+    private int REMEMBER_ME_TIMEOUT;
+
+    @Value("${spring.security.remember-me.token:#{rememberMeTokenGenerator}}")
```

Comment:
Is this really a well known / standard property for configuring this? I can't find any docs on this. If not I'd suggest to remove this configuration option...

Reply:
I have removed that, since it is not a standard property.